projectcard/examples/transit-route_addition.yml

```diff
@@ -0,0 +1,45 @@
+project: New Green Transit
+tags:
+   - new transit
+transit_route_addition:
+  routes:
+    - route_id: abc
+      route_long_name: green_line
+      route_short_name: green
+      route_type: 3
+      agency_id: The Bus
+      trips:
+        - direction_id: 0
+          headway_secs:
+            ('6:00','12:00'): 600
+            ('12:00','13:00'): 900
+          routing:
+            - 1:
+                stop: true #when stop is set to True, defaults to allow both boarding and alighting
+            - 2
+            - 3
+            - 4:
+                stop: true # default to False, specify only when stopping
+                alight: false  # default to True, specify only when not default
+            - 5
+            - 6:
+                stop: true
```

Comment:
stop ids should be strings

Reply:
they are read in as strings - unless you had issues with that?

Reply:
git it, I thought the example would be 
- "1":
- "2":
this is how the transit_route_addition project is written out by Cube/Metcouncil Wrangler

Reply:
that should be ok - they are basically the same thing - YAML defaults to string when it is supposed to be string and doesn't require encapsulating in quotes.  When it is read in and validated, it knows that it is supposed to be a string so it parses it as that.